files/build_templates/per_namespace/teamd.service.j2

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=TEAMD container
+Requires=updategraph.service
+After=swss{% if multi_instance == 'true' %}@%i{% endif %}.service
+After=updategraph.service
```

Comment:
Suggest to move this line under line 3 to keep it together with the `Requires=updategraph.service` line.

Reply:
updated as per comment.